src/main/java/org/frc5687/chargedup/subsystems/ExtendingArm.java

```diff
@@ -0,0 +1,91 @@
+package org.frc5687.chargedup.subsystems;
+
+import org.frc5687.chargedup.Constants;
+import org.frc5687.chargedup.util.OutliersContainer;
+import org.frc5687.lib.drivers.OutliersTalon;
+import org.frc5687.lib.sensors.HallEffect;
+import org.frc5687.chargedup.RobotMap;
+
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+
+import edu.wpi.first.math.controller.PIDController;
+
+public class ExtendingArm extends OutliersSubsystem {
```

Comment:
I think ExtendingArm and Arm subsystems are too close together in the name. We could call It the Elevator or something else but it's not a major issue.